### ISSUE TITLE
[Gujarati] Invalid cluster example

### ISSUE THREAD

**Issue by kalapi** [n8willis/opentype-shaping-documents#1]
Hello! This is a great resource, thanks and congratulations!

The following cluster would never occur:

<img width="312" alt="screen shot 2018-02-10 at 7 33 44 am" src="https://user-images.githubusercontent.com/15804460/36057500-eb045502-0e34-11e8-8105-402e9f528784.png">

You can use the following instead (text is ર્હ ):

<img width="81" alt="screen shot 2018-02-10 at 7 33 10 am" src="https://user-images.githubusercontent.com/15804460/36057504-f708cefa-0e34-11e8-8659-eb88adf81ee6.png"><img width="52" alt="screen shot 2018-02-10 at 7 33 23 am" src="https://user-images.githubusercontent.com/15804460/36057506-fb36eb9c-0e34-11e8-8e15-3843107f5629.png">

Best,
Kalapi

**Comment by n8willis** [comment]
Thanks! Just to capture for later reference, you're referring to the `abvm` illustration at https://github.com/n8willis/opentype-shaping-documents/blob/master/opentype-shaping-gujarati.md#6-applying-remaining-positioning-features-from-gpos -- right?

Also I'm assuming that your replacement is also made in Noto Gurajarati. Can you tell me the Unicode codepoints? I ask because I was generating all images from a hopefully-reproducible hb-view sequence; in this case the old one is here: https://github.com/n8willis/opentype-shaping-documents/blob/master/images/gujarati/gujarati-image-generation-log.md#6-abvm

Keeping them recorded this way seemed better for long-term change tracking....

**Comment by n8willis** [comment]
Also, for the record, is the cluster in the existing image impossible, or just not useful? In most cases, I only have the GSUB lookups in the font to go on, so if the font includes one that isn't possible (as in this case), it would be nice to know. [The lookup table might have been bulk-generated in the original font; I just don't know]

**Comment by n8willis** [comment]
@kalapi Ping? I could genuinely use some clarifications about the underlying issues here. :)

**Comment by kalapi** [comment]
Sorry @n8willis I didn't see this earlier (I'm in a deluge of issue notifications from participating in other repos :)

1. Yes indeed, the new illustration is suggested as a replacement for the `abvm` example.
2. Yes, it is indeed generated using Noto Serif Gujarati. The following replacement can be used:

```
hb-view --font-size=110 --margin=2,16,2,16 --output-file=gujarati-abvm-before.png --features=-init,-abvm --background=FFFFFF00 /usr/share/fonts/truetype/noto/NotoSerifGujarati-Regular.ttf --unicodes=0ab0,0acd,0ab9
```
3. The cluster is not _impossible_ per say, but it is unpronounceable and thus will never occur. It is included in the font because, during development, the _reph_ sign shares the above anchor with modifiers such as the _anusvara_ and the _candrabindu_, and thus ends up as part of the coverage table.

**Comment by n8willis** [comment]
No problem! Thanks for the info.

This should be fixed by 86bfe7998056eeafe57b52207ac469efa53f691d .
